drivers/disk/flashdisk.c

```diff
@@ -19,6 +19,11 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(flashdisk, CONFIG_FLASHDISK_LOG_LEVEL);
 
+#if defined(CONFIG_FLASH_HAS_EXPLICIT_ERASE) &&	\
+	defined(CONFIG_FLASH_HAS_EXPLICIT_ERASE)
```

Comment:
Duplicate checks?

Reply:
You can never be sure what these pesky Kconfigs are doing, better check twice.
Fixed.